tests/lxd/test_lxd_backend.cpp

```diff
@@ -256,6 +260,57 @@ TEST_F(LXDBackend, machine_persists_and_sets_state_on_shutdown)
     EXPECT_EQ(machine.current_state(), mp::VirtualMachine::State::stopped);
 }
 
+TEST_F(LXDBackend, machine_does_not_update_state_in_dtor)
+{
+    NiceMock<mpt::MockVMStatusMonitor> mock_monitor;
+
+    bool vm_shutdown{false};
+
+    ON_CALL(*mock_network_access_manager.get(), createRequest(_, _, _))
+        .WillByDefault([&vm_shutdown](auto, auto request, auto outgoingData) {
+            outgoingData->open(QIODevice::ReadOnly);
+            auto data = outgoingData->readAll();
+            auto op = request.attribute(QNetworkRequest::CustomVerbAttribute).toString();
+            auto url = request.url().toString();
+
+            if (op == "GET")
+            {
+                if (url.contains("1.0/operations/b043d632-5c48-44b3-983c-a25660d61164"))
+                {
+                    vm_shutdown = true;
```

Comment:
Is this actually `vm_shutdown`? And not… `waiting`?

Reply:
So this is just the `bool` to say that the client requested the `GET` of the `operations` URL and that we dictated   the VM did indeed fully shut down.  It's basically used to indicate that the client went through all of its logic to know that the vm is fully shut down.

Reply:
Oh right, the return below says the operation completed, which means the VM is shut down. My bad.